### ISSUE TITLE
Is there a way to make it's position relative?

### ISSUE THREAD

**Issue by jorexe** [localz/react-native-searchbar#13]
I want to use the search bar inside a view this way:
```javascript
<SearchBar ref={(ref) => this.searchBar = ref}
	data={this.props.data}
	handleResults={(results) => this.setState({results})}
	showOnLoad
/>
<Results />
```

**Comment by daveols** [comment]
This isn't possible at the moment unfortunately @jorexe.

If you're able, fork the code and implement it! If there is more interest I may consider doing it myself.

**Comment by jneuendorf** [comment]
Hey, I forked the repo and committed this: https://github.com/jneuendorf/react-native-searchbar/commit/c869b61d9ad322628d7c45e9476d6c42d8d775f5
This gives the user full flexibility over all styles. I started playing around with it and just setting `position: relative` messes up the layout.
Maybe one could add another prop that use a different style preset than the current one...

@daveols What do you think?
Should I open a PR?

**Comment by daveols** [comment]
Thanks @jneuendorf! Looks like your approach may work but it doesn't take into account the behaviour of the drop down animation. It'd also be up to the user to implement the relative styles correctly -- would much rather bake this in or switch mode as you mentioned!

**Comment by dviluk** [comment]
more easy if you let us modify the style from props

**Comment by jneuendorf** [comment]
That's true I guess. But I was working on the 'baked in' version at work. Currently, it's still kinda quickly and hard-coded and I don't have that much more time to make it more generic in order to open a PR.
But I will once I got to it 😉
